packages/components/tests/dummy/app/templates/components/table.hbs

```diff
@@ -3,17 +3,7 @@
   Table
 </h2>
 <p class="dummy-paragraph">
-  This table component is the
-  <a
-    href="https://unofficial-design-system-search.vercel.app/?path=/docs/cloud-ui-storybook_table--static"
-    rel="noopener noreferrer"
-    target="_blank"
-  >Cloud-UI
-    <code class="dummy-code">&lt;Table&gt;</code>
-    component</a>
-  with a few adjustments for improved WCAG conformance. Users who already have this table component implemented should
-  be able to migrate to this component with little to no difficulty. Additionally, the data-test attributes have been
-  left "as is" to avoid breaking your existing tests.
+  This table component is a simple data table that supports using a data model...or not, to keep the use case flexible.
```

Comment:
[heads-up] also the "Overview" section should be updated to reflect the changes in this PR